docs/development/how_to_contribute.md

```diff
@@ -1,4 +1,184 @@
 
+**Thanks for your interest in the GraphScope project.**
+
+GraphScope is a large project and may seem overwhelming when you’re first getting involved. Contributing code is great,
+but that’s probably not the first place to start. There are lots of ways to make valuable contributions to the project and community, 
+from improving the documentation, submitting bug reports and feature requests or writing code which can be incorporated into GraphAr itself.
+
+## Code of Conduct
+
+Please read our [Code of Conduct][1] before contributing or engaging in our community.
+
+## Community
+
+A good first step to getting involved in the GraphScope project is to join us and participate in discussions where you can. there are several ways to connect with us.
+
+### GitHub Discussion
+
+We use [GitHub Discussion](https://github.com/alibaba/GraphScope/discussions) as a way to ask questions. Feel free to ask qustion if you have questions about
+GraphScope or issues with your own code.
+
+### Slack
+
 TBF
-(This document is intended to help you get started contributing to GraphScope project.)
\ No newline at end of file
+
+### DingTalk
+
+TBF
+
+## Bug Report
+
+If you find a bug in GraphAr, first make sure that you are testing against the [latest version of GraphScope](https://github.com/alibaba/GraphScope/tree/main), 
+your issue may already have been fixed. If not, search our [issues list](https://github.com/alibaba/GraphScope/issues)
+on GitHub in case a similar issue has already been opened.
+
+If you get confirmation of your bug, [file a bug issue](https://github.com/alibaba/GraphScope/issues/new?assignees=&labels=&template=bug_report.md&title=%5BBUG%5D+) before starting to code. When submitting an issue, please include a clear and concise description of the problem, any relevant code or error messages, and steps to reproduce the issue.
+
+## Request Feature
+
+If you find yourself wishing for a feature that doesn't exist in GraphScope, you are probably not alone.
+There are bound to be others out there with similar needs. Many of the features that GraphScope has today
+have been added because our users saw the need.
+
+[Open a feature request issue](https://github.com/alibaba/GraphScope/issues/new?assignees=&labels=&template=feature_request.md&title=) on GitHub which describes the feature you would
+like to see, why you need it, and how it should work.
+
+## Improve Documentation
+
+A great way to contribute to the project is to improve documentation. If you found some docs to be incomplete or inaccurate, share your hard-earned knowledge with the rest of the community.
+
+Documentation improvements are also a great way to gain some experience with our submission and review process, discussed below, without requiring a lot of local development environment setup. In fact, many documentation-only changes can be made directly in the GitHub web interface by clicking the “edit” button. This will handle making a fork and a pull request for you.
```

Comment:
in the Github web? no, we have a more straight-forward way.  we have the edit button on the docs page.